.github/workflows/integration_test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Run docker in background
         run: |
-          docker run --detach --rm --name rippled-service -p 5005:5005 -p 6006:6006 --volume "${{ github.workspace }}/.ci-config/":"/opt/ripple/etc/" --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s --env GITHUB_ACTIONS=true --env CI=true ${{ env.RIPPLED_DOCKER_IMAGE }} /opt/ripple/bin/rippled -a --conf /opt/ripple/etc/rippled.cfg
```

Comment:
Why all the changes here?

Reply:
- I have updated the docker image to use the latest tip from `develop` branch
- I'm given to understand that there are some updates to the `entrypoint.sh` bash file in the docker image. In consultation with the @micheal legleux, I've updated the `docker run` command accordingly.

Reply:
lastly, I've updated the mount point of the docker container. We don't need to explicitly specify the conf file anymore, its already placed into the default location that is searched by rippled

Reply:
Some of this is now just moving some fields around, which IMO should be undone. Also, why is the mount point changing?

Reply:
the `entrypoint` in the docker file has been updated. These changes accomodate the new docker-file. 

One of the advantages of the new mount point is that, we don't need to specify the path to `rippled.cfg` file explicitly.